tutorials/Tutorial2_Finetune_a_model_on_your_data.py

```diff
@@ -47,6 +47,45 @@ def tutorial2_finetune_a_model_on_your_data():
     # If you want to load it at a later point, just do:
     new_reader = FARMReader(model_name_or_path="my_model")
 
+    # ## Distill your model
+    # In this case, we have used "distilbert-base-uncased" as our base model.
+    # This model was trained using a process called distillation.
+    # In this process, a bigger model is trained first and is used to train a smaller model which increases its accuracy.
+    # This is why "distilbert-base-uncased" can achieve quite competitive performance while being very small.
+    #
+    # Sometimes, however, you can't use an already distilled model and have to distil it yourself.
+    # For this case, haystack has implemented a [distillation approach](https://arxiv.org/pdf/1909.10351.pdf).
+    # distil(reader)
+
+
+def distil(teacher):
+    # ### Augmenting your training data
+    # To get the most out of model distillation, we recommend increasing the size of your training data by using data augmentation.
+    # You can do this by running the [`augment_squad.py` script](https://github.com/deepset-ai/haystack/blob/master/haystack/utils/augment_squad.py):
+    # # Downloading script
+    # !wget https://raw.githubusercontent.com/deepset-ai/haystack/master/haystack/utils/augment_squad.py
+    # # Just replace dataset.json with the name of your dataset and adjust the output path
+    # !python augment_squad.py --squad_path dataset.json --output_path augmented_dataset.json --multiplication_factor 2
```

Comment:
Is there a reason why you decided against calling the script here while calling it in the jupyter notebook? Could you please try calling the script here as well? Even if we need to make some small changes to the augment_squad.py script it would be good to call the script from within the code here.

Reply:
There isn't really a good reason except that it required to make one change in `augment_squad.py`. However, I have now made this change and the script is now called from the tutorial.